File: inspect_excel.js
const ExcelJS = require('exceljs');
const path = require('path');

async function inspectExcel() {
    const workbook = new ExcelJS.Workbook();
    const filePath = path.join('f:', 'ss', 'ss', '아르카나 이벤트 이름 도움.xlsx');

    try {
        await workbook.xlsx.readFile(filePath);
        const worksheet = workbook.getWorksheet(1); // Get first sheet

        console.log('Sheet Name:', worksheet.name);
        console.log('Row Count:', worksheet.rowCount);

        // Print first 5 rows
        worksheet.eachRow((row, rowNumber) => {
            if (rowNumber <= 5) {
                console.log(`Row ${rowNumber}:`, row.values);
            }
        });
    } catch (error) {
        console.error('Error reading Excel file:', error);
    }
}

inspectExcel();

File: inspect_excel.py
import pandas as pd

file_path = '스타 세이비어 아르카나 V1.0_251125의 사본.xlsx'

try:
    # Load the Excel file to list sheet names
    xls = pd.ExcelFile(file_path)
    print(f"Sheet names: {xls.sheet_names}")

    # Read the 'DB' sheet if it exists, otherwise read the first sheet
    if 'DB' in xls.sheet_names:
        df = pd.read_excel(file_path, sheet_name='DB')
        print("\nColumns in 'DB' sheet:")
        print(df.columns.tolist())
        print("\nFirst 3 rows:")
        print(df.head(3))
    else:
        print("\n'DB' sheet not found.")
except Exception as e:
    print(f"Error: {e}")
